galata/src/helpers/notebook.ts

```diff
@@ -1414,16 +1414,36 @@ export class NotebookHelper {
   }
 
   /**
-   * Create a new notebook
+   * Creates a new notebook.
    *
-   * @param name Name of the notebook
-   * @returns Name of the created notebook or null if it failed
+   * @param name - The name of the notebook. If provided, the notebook will be renamed to this name.
+   * @param options - Parameters for creating the notebook.
+   * @param options.kernel - The kernel to use for the notebook.
+   * @returns A Promise that resolves to the name of the created notebook, or `null` if the notebook creation failed.
    */
-  async createNew(name?: string): Promise<string | null> {
+  async createNew(
+    name?: string,
+    options?: { kernel?: string }
+  ): Promise<string | null> {
     await this.menu.clickMenuItem('File>New>Notebook');
 
     const page = this.page;
     await page.locator('.jp-Dialog').waitFor();
+
+    if (options?.kernel) {
+      if (options.kernel === 'null') {
```

Comment:
Would it make sense to make it an actual `null` value here?

```suggestion
    options?: { kernel?: string | null }
  ): Promise<string | null> {
    await this.menu.clickMenuItem('File>New>Notebook');

    const page = this.page;
    await page.locator('.jp-Dialog').waitFor();

    if (options?.kernel) {
      if (options.kernel === null) {
```

Reply:
I wouldn't have expected this to work, as `null` is Falsy, so we wouldn't end up inside the first if block. 

There would be no difference between setting it to null and not setting it at all. As I'm trying to make this backwards compatible, I would rather set it to something like "No Kernel".

Reply:
`null` and `undefined` are different values in JavaScript and differentiated when using triple equals (`===`).

Reply:
Sure, but the `if (options?kernel)` check would fail for null, right?

Reply:
Double checked and this is backwards compatible with `null`. Let's go with using `null` as it is being used like this in other public APIs.

Reply:
I don't think we would be able to set the No Kernel scenario with code in its current state - I think we need to modify the first if statement I proposed. 
We would need to explicitly check that options.kernel is not undefined now. But that should then work